pkg/infra/pulumi_aws/iac/rds.ts

```diff
@@ -0,0 +1,215 @@
+import * as aws from '@pulumi/aws'
+import * as pulumi from '@pulumi/pulumi'
+import { hash as h, sanitized, validate } from './sanitization/sanitizer'
+import AwsSanitizer from './sanitization/aws'
+import { Resource, CloudCCLib, RdsImport, kloConfig } from '../deploylib'
+
+export interface CreateInstanceAndProxyResult {
+    rds: aws.rds.Instance
+    proxy: aws.rds.Proxy
+}
+
+export class RDS {
+    constructor() {}
+
+    static setupRDS(lib: CloudCCLib, orm: string, args: Partial<aws.rds.InstanceArgs>) {
+        const rdsImports: { [key: string]: RdsImport } | undefined = kloConfig.getObject('rds')
+        const config = new pulumi.Config()
+        const dbName = sanitized(
+            AwsSanitizer.RDS.engine.pg.database.nameValidation()
+        )`${orm.toLowerCase()}`
+        const username = config.require(`${dbName}_username`)
+        const password = config.requireSecret(`${dbName}_password`)
+
+        let rds
+        let proxy
+
+        if (rdsImports != undefined && rdsImports[orm] != undefined) {
```

Comment:
should we also check for VPC here if required?

Reply:
we dont have to, if they want a separate vpc for their workloads i think we should allow it, it would just be on them to peer their vpcs.

If we want to just because were saying its the only use case we support today then i guess we could, im torn on if thats a great idea or not though

Reply:
Ah that's fair, this is fine for now.

Reply:
especially because in the lambda case it could still connect, its mainly just for the ecs and eks case. So i dont think we want that check

Reply:
ah nevermind it still wouldnt be able to connect because of the Security Group